### DISCUSSION TITLE
XFormers bad version

### DISCUSSION THREAD

**Discussion by dymension** [TheLastBen/fast-stable-diffusion#2789] [General]
Hello Ben,

From tonight, when I laucnh A111 in Google Collab, I have this message error when starting the Stable Diffusion card

```
WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.1.0+cu121 with CUDA 1202 (you have 2.2.1+cu121)
    Python  3.10.12 (you have 3.10.12)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details
```

It looks like a version conflict
 xFormers was built for:     **PyTorch 2.1.0**+cu121 with CUDA 1202 (**you have 2.2.1**+cu121)

How can we fix that ?

**Comment by dymension** [comment]
Solution from Maxim :

> You need to insert this text into "Connect Google Drive": (after the `#@markdown ---` in the code)
> 
> ```
> !pip install lmdb
> !pip install torch==2.1.0+cu118 torchvision==0.16.0+cu118 torchaudio==2.1.0 torchtext==0.16.0+cpu torchdata==0.7.0 --index-url https://download.pytorch.org/whl/cu118
> ```
>

**Comment by dymension** [comment]
![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/1997716/276966e0-ccb3-487f-a355-88ac4f06af45)

**Comment by TheLastBen** [comment]
fixed, update to the latest notebook and you won't need to install anything

**Reply by dymension** [reply]
Thanks for solving the issue !!
